packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.types.ts

```diff
@@ -63,24 +60,36 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    */
   directionalHintForRTL?: DirectionalHint;
 
+  /**
+   * Display property to be applied to the host wrapper div.
+   * When wrapping elements which have display 'inline' or 'inline-block' there are instances when Tooltip covers the target when rendered.
+   * This happens due to default value 'inline' of the wrapper which computes it's line-height from the inherited font-size and being placed
+   * at the base line of the wrapped element. If line-heights of both don't match it will render not in the intended position.
+   * @default 'inline'
+   */
+  display?: 'inline' | 'inline-block' | 'block';
```

Comment:
is it possible to use this and update it to include other display values (flex etc) https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/merge-styles/src/IRawStyleBase.ts#L794

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/display

Reply:
Should I update this too: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/merge-styles/api/merge-styles.api.ts#L111

Reply:
@dzearing is this merge-styles.api.ts file an auto generated file?